lib/pill.hoon

```diff
@@ -0,0 +1,122 @@
+::  |pill: helper functions for making pills
+::
+|%
+::  +module-ova: vane load operations.
+::
+::    sys: full path to /sys directory
+::
+++  module-ova
+  |=  sys=path
+  ^-  (list ovum)
+  |^  :~  ::  sys/zuse: standard library
+          ::
+          (vent %$ /zuse)
+          ::  sys/vane/ames: network
+          ::
+          (vent %a /vane/ames)
+          ::  sys/vane/behn: timer
+          ::
+          (vent %b /vane/behn)
+          ::  sys/vane/clay: revision control
+          ::
+          (vent %c /vane/clay)
+          ::  sys/vane/dill: console
+          ::
+          (vent %d /vane/dill)
+          ::  sys/vane/eyre: web
+          ::
+          (vent %e /vane/eyre)
+          ::  sys/vane/ford: build
+          ::
+          (vent %f /vane/ford)
+          ::  sys/vane/gall: applications
+          ::
+          (vent %g /vane/gall)
+          ::  sys/vane/jael: security
+          ::
+          (vent %j /vane/jael)
+      ==
+  ++  vent
+    |=  [abr=term den=path]
+    ^-  ovum
+    =/  pax  (weld sys den)
+    =/  txt  .^(@ %cx (weld pax /hoon))
+    [[%vane den] [%veer abr pax txt]]
+  --
+::  +file-ovum: userspace filesystem load
+::
+::     bas: full path to / directory
+::
+++  file-ovum
+  |=  bas=path
+  ^-  ovum
+  ::
+  ::    /app    %gall applications
+  ::    /gen    :dojo generators
+  ::    /lib    %ford libraries
+  ::    /mar    %ford marks
+  ::    /ren    %ford renderers
+  ::    /sec    %eyre security drivers
+  ::    /sur    %ford structures
+  ::    /sys    system files
+  ::    /tests  unit tests
+  ::    /web    %eyre web content
+  ::
+  %.  [/app /gen /lib /mar /ren /sec /sur /sys /tests /web ~]
```

Comment:
Is there a reason we enumerate these instead of just doing a `.^(arch %cy path)`?

It might also be worth considering using the `%ct` Clay scry to get a full recursive file listing instead of manually recursing here.

Reply:
This is just copying around the original implementation for cc-release pills -- first in `+metal`, later in `+brass`. Maybe it was for documentary reasons the first time? I'm not opposed to refactoring and improving it, but it does work, and reasonably quickly.

Reply:
Ok. We can clean it up in the glorious future after everything else works perfectly.